spec/neat/pad_spec.rb

```diff
@@ -25,7 +25,29 @@
 
   context "with argument (default)" do
     it "uses default gutter percentage" do
-      expect(".pad-shorthand-default").to have_rule("padding: 30px 2.35765% 10px 2.35765%")
+      expect(".pad-shorthand-default").to have_rule("padding: 30px 2.35765% 10px")
+    end
+  end
+
+  context "with argument (null)" do
+    it "ignores rules with null values" do
+      ruleset = "padding-right: 10px; " +
+                "padding-left: 10px;"
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.